data/questions_gen_postgres.csv

```diff
@@ -95,7 +95,7 @@ Filter state code (eg NY), airport codes (eg JFK) using case-insensitive matches
 If multiple flight days are requested, use ILIKE for each of the days separately, since they are not necessarily ordered."
 What is the total cost of all round-trip fares from New York (JFK) to Los Angeles?,SELECT SUM(fare.round_trip_cost) AS total_round_trip_cost FROM fare WHERE fare.from_airport = 'JFK' AND fare.to_airport = 'LAX';,atis,instruct,"Filter airport, city, country names using ILIKE.
 Filter state code (eg NY), airport codes (eg JFK) using case-insensitive matches.
-If multiple flight days are requested, use ILIKE for each of the days separately, since they are not necessarily ordered."
+fare.round_trip_required is not needed when getting the round trip cost."

```

Comment:
Specify an additional instruction so that the model will know to not filter `fare.round_trip_required='Yes'`, since the 2 columns dealing with round trips are ambiguously related to the question.